learning/src/components/WordDefinition.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import { render } from "react-dom";
+
+function SoundButton(props) {
+    return (
+        <div>
+            <i 
+                className="fas fa-volume"
+                id="sound-button"
+                onClick={() => document.getElementById("audio").play()}
+            ></i>
+            <audio id="audio">
+                <source src={props.src} type="audio/mpeg"></source>
+            </audio>
+        </div>
+    );
+}
+
+function WordPinyin(props) {
+    return (
+        <ruby>
+            {props.word}
+            <rp>(</rp>
+            <rt>{props.pinyin}</rt>
+            <rp>)</rp>
+        </ruby>
+    );
+}
+
+class Definitions extends Component {
```

Comment:
For naming, add the prefix `word` if it will be used in the word display. This avoids namespace collision since we are also doing char and radical displays too.